handlers/userHandler.js

```diff
@@ -0,0 +1,64 @@
+const User = require("../model/userModel");
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const Logging = require('../library/Logging');
```

Comment:
```suggestion
const Logger = require('../library/Logger');
```

Please instead of Logging use the word Logger. Please check all places and apply change accordingly